src/code/z_debug_display.c

```diff
@@ -88,7 +88,7 @@ void DebugDisplay_DrawSpriteI8(DebugDispObject* dispObj, u32 texture, GlobalCont
     CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_debug_display.c", 192);
 }
 
-void DebugDisplay_DrawPolygon(DebugDispObject* dispObj, u32 dlist, GlobalContext* globalCtx) {
+void DebugDisplay_DrawPolygon(DebugDispObject* dispObj, void* dlist, GlobalContext* globalCtx) {
```

Comment:
So `Gfx*` would be better but I assume we prefer `void*` to match the `DebugDispObject_DrawFunc` function pointer type, to avoid casting the function in `sDebugObjectDrawFuncTable`

Reply:
Yeah. Unless there is a way to have both